src/app/space/create/deployments/apps/deployment-card.component.ts

```diff
@@ -13,6 +13,20 @@ import { CpuStat } from '../models/cpu-stat';
 import { Environment } from '../models/environment';
 import { DeploymentsService } from '../services/deployments.service';
 
+export const CLASSES = Object.freeze({
+  ICON_OK: 'pficon-ok',
+  ICON_WARN: 'pficon-warning-triangle-o',
+  ICON_ERR: 'pficon-error-circle-o'
+});
```

Comment:
Why use a frozen object rather than an actual Enum?

https://www.typescriptlang.org/docs/handbook/enums.html

And, rather than exporting this value separately in the file, how about moving it inside of the component class? Then it doesn't require an additional import for consumers of the component, and the intention/purpose of the values is more obvious at a glance. Also, as Jie said, these enums should live in the icon component class, not its parent. The idea is that there could be various, even potentially unknown third party, parents to the icon, but the icon should expect to only be used with some specific set of values that it already knows about.